src/index.ts

```diff
@@ -103,11 +93,9 @@ async function _onNewBidiConnectionOpen(
     await mapperServer.sendMessage(message);
   });
 
-  // Save handler `closeBrowser` to use after the client disconnected.
-  return browserProcess;
-}
-
-function _onBidiConnectionClosed(browserProcess: BrowserProcess): void {
-  // Client disconnected. Close browser.
-  browserProcess.closeBrowser();
+  // Return delegate to be called when the connection is closed.
+  return async () => {
+    // Client disconnected. Close browser.
+    await browser.close();
+  };
```

Comment:
`return browser.close()`?

Reply:
`return browser.close`, without brackets, but it can be confusing, and can be read as `close browser and return`, while it should be `return delegate, calling which will close the browser`.